### PR TITLE
app-text/cmark: bump to 0.28.0

### PR DESCRIPTION
**Pull request by winterheart**
Package-Manager: Portage-2.3.6, Repoman-2.3.1

### CODE REVIEW

File: app-text/cmark/cmark-0.28.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_{4,5} )
+
+inherit cmake-utils python-any-r1
+
+DESCRIPTION="CommonMark parsing and rendering library and program in C"
+HOMEPAGE="https://github.com/jgm/cmark"
+SRC_URI="https://github.com/jgm/cmark/archive/${PV}.tar.gz -> ${P}.tar.gz"
```
Comment (mgorny) [inline app-text/cmark/cmark-0.28.0.ebuild]:
```
HTTP request sent, awaiting response... 301 Moved Permanently
Location: https://github.com/commonmark/cmark/archive/0.28.0.tar.gz [following]
```

You need to update the URLs.

File: app-text/cmark/cmark-0.28.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_{4,5} )
```
Comment (mgorny) [inline app-text/cmark/cmark-0.28.0.ebuild]: Oh, and add 3.6 here, it seems to work fine for me.

**Comment by winterheart** [comment]
@mgorny done.

**Comment by mgorny** [comment]
The entry in metadata.xml needs fixing too, but I'll do that for you.

**Pull request closed without merging**